backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -358,21 +279,22 @@ private Entity.Parcel AddUpdateParcel(Model.ImportPropertyModel property, int pi
                 p_e.ClassificationId = propClassification.Id;
                 p_e.Classification = propClassification;
 
-                var city = GetOrCreateCity(property.City);
+                // TODO: Handle this issue more gracefully.
+                var city = _pimsAdminService.AdministrativeArea.Get(property.City.ConvertToUTF8()) ?? throw new InvalidOperationException($"Administrative area '{property.City}' does not exist in the datasource.");
```

Comment:
This will cause developer issues when they use the `make db-seed` command because their data will have cities that don't exist in the `Administrative Areas`.  Just temporarily update your seed data to use only cities we have.